torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -106,6 +138,12 @@ class ProcessGroup {
     std::condition_variable cv_;
     bool completed_ = false;
     std::exception_ptr exception_;
+
+    // Current rank of the node.
```

Comment:
nit: maybe better to use process/thread instead of node. In distributed context, I would think node represents a machine.